manifests/editors/atom.pp

```diff
@@ -52,6 +52,23 @@
         require => Package['Atom'],
       }
     }
+    'Debian': {
+      validate_string($url)
+      $apm_require = Package['atom']
+
+      archive { 'atom-amd64.deb':
+        source  => 'https://atom.io/download/deb',
+        path    => '/tmp/atom-amd64.deb',
+        extract => false,
+        cleanup => false,
```

Comment:
Hmm I need to investigate this more.  I have seen examples of downloading debs before...
Not so sure we need to keep downloading it to /tmp once it is installed. I think some other solutions I have seen solve that somehow. I need to go digging

Reply:
Also in puppet 4 the file type can take http urls as source now. If there is no extraction, we may want to use that to reduce dependencies